features/mirror_spec.rb

```diff
@@ -0,0 +1,8 @@
+require File.expand_path('../support/command_rspec_helper', __FILE__)
+
+describe 'mirror' do
+  before { system '/usr/bin/rmt-cli repos enable 3114' }
+
+  command '/usr/bin/rmt-cli mirror'
+  its(:exitstatus) { is_expected.to eq 0 }
```

Comment:
Mirroring process would exit with code zero if mirroring of some repositories failed. We aren't really checking if we managed to download any RPM files.

Reply:
good point! fixed